bin.js

```diff
@@ -30,35 +31,19 @@ function build () {
 
   proc.spawn(args[0], args.slice(1), { stdio: 'inherit', shell: win32, windowsHide: true }).on('exit', function (code) {
     if (code || !process.argv[3]) process.exit(code)
-    exec(process.argv[3]).on('exit', function (code) {
-      process.exit(code)
-    })
   })
 }
 
 function preinstall () {
-  if (!process.argv[2]) return build()
-  exec(process.argv[2]).on('exit', function (code) {
-    if (code) process.exit(code)
+  try {
+    // try to load the prebuild
+    const load = require(path.join(__dirname, 'index.js'))
+    load()
+  } catch (err) {
+    // report the error and fall to a build
+    console.error(err.message)
```

Comment:
I don't think this should be an error. That would imply the consuming package has done something wrong or has an unexpected condition to recover from.

More likely in my experience, the package was simply not built.